src/flows/pipes/Columns/index.tsx

```diff
@@ -0,0 +1,57 @@
+import View, {TableColumnKey} from './view'
+
+export default register => {
+  register({
+    type: 'columnEditor',
+    family: 'passThrough',
+    component: View,
+    button: 'Column Editor',
+    initial: {
+      updatedTableKeys: {},
+    },
+    generateFlux: (pipe, create, append) => {
+      append(`__CURRENT_RESULT__ |> limit(n: 100)`)
+      if (!Object.values(pipe.updatedTableKeys).length) {
+        return
+      }
+
+      const toggleHide = Object.entries(
+        pipe.updatedTableKeys as {[_: string]: TableColumnKey}
```

Comment:
why are we typecasting this?

Reply:
bc we need to have a type for the pipe.updatedTableKeys